symbulate/distributions.py

```diff
@@ -299,6 +299,35 @@ def draw(self):
 
         return np.random.poisson(lam=self.lam)
 
+class DiscreteUniform(Distribution):
+    """Defines a probability space for a discrete uniform distribution.
+
+    Attributes:
+      a (int): lower bound for possible values
+      b (int): upper bound for possible values
+    """
+
+    def __init__(self, a=0, b=1):
+        self.a = a
+        self.b = b
+        
+        params = {
+            "low" : self.a,
+            "high" : self.b
+            }
+
+        if (b-a) <= 0:
```

Comment:
Prefer `if a > b:`. You should allow a = b. (It would just always return the value a=b.)